src/main/java/frc/trigon/robot/components/XboxController.java

```diff
@@ -2,8 +2,12 @@
 
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.StartEndCommand;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
-import frc.trigon.robot.commands.Commands;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+import static edu.wpi.first.wpilibj.GenericHID.RumbleType.kBothRumble;
```

Comment:
Static imports are ugly. If I see something that's staticly imported I'd assume it's from the current viewed class until I'll scroll up to find that it's staticly imported. Just qualify the class itself to reference where it's from in the code. Remove this static import.